src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -458,16 +458,6 @@ public void configureWebApplication() throws Exception {
         
         super.configureWebApplication();
         getWebAppConfig().setWar(webAppFile.getCanonicalPath());
-        for (Artifact a : (Set<Artifact>) project.getArtifacts()) {
-            if (a.getGroupId().equals("org.jenkins-ci.main") && a.getArtifactId().equals("jenkins-core")) {
-                File coreBasedir = pluginWorkspaceMap.read(a.getId());
-                if (coreBasedir != null) {
-                    String extraCP = new File(coreBasedir, "src/main/resources").toURI() + "," + new File(coreBasedir, "target/classes").toURI();
```

Comment:
So IIUC this is removing a fairly important aspect of `hudson-dev:run`. Without this, we would have to restart Jenkins every time we edited any resource.